lib/guardian/channel.ex

```diff
@@ -37,14 +37,13 @@ defmodule Guardian.Channel do
   directly and authenticating the connection rather than the channel.
   """
   defmacro __using__(opts) do
-    opts = Enum.into(opts, %{})
-    key = Map.get(opts, :key, :default)
+    key = Keyword.get(opts, :key, :default)
```

Comment:
@hassox I made this change because, at first glance, it didn't look like `opts` was being used other than retrieving `key` and converting to a map seemed like overkill.